### ISSUE TITLE
To do 11/9

### ISSUE THREAD

**Issue by srkobakian** [srkobakian/review#3]
- [x] Create a 2x2 display of USA (all except choropleths), is there data / polygons for Hawaii and Alaska?
- [x] Choose user friendly colour schemes for Aus and USA
- [x] Make figures full width
- [x] Add in Australia examples (same display as USA) (SA3 example: http://theconversation.com/interactive-we-mapped-cancer-rates-across-australia-search-for-your-postcode-here-102256)
- [x] Add citation for USA data download
- [x] Rethink section layers
- [x] Refer to section numbers
- [x] Subsections 3.1.1, 3.1.2, ... for each of the publicly atlases: country/region, statistic, year, **Jessie**

- [x] Use Australia as a reason to explore alternative approaches (user feedback by Earl)
- [x] Include Bob Rudis heatmap display (equal area tilegrams)
- [x] Mention all packages in acknowledgements (proper package citations)
- [x] Mention animation in practice (gganimate pres/paper?, Google Earth Dr Jane Goodall story)
- [x] Subsections 3.3.1, 3.132, ... for each of the publicly atlases, summarise the interaction capabilities **Jessie**
- [ ] Conclusions section **Di**
- [ ] Copy-edit section 3 **Di**
- [ ] Copy-edit section 4 **Di**
- [x]  add citations to each map description. **Jessie**
- [x] find map with time animation for Steph ** Jessie**

**Comment by jesse-jesse** [comment]
YES!!! @srkobakian I found 1 - http://www.geeitema.org/AtlasET/atlas.jsp?causa=e27MEPOC 
Takes ages to load. You have to select a cancer type bfore you can see the map

I can only find one. I think the other one is within one of the flash maps. which I have to update flash. Do you think we need more than one? I only remember 2 .. 

i also found this one which is an interesting type of interaction.. kind of magnifying glass. http://www.ehatlas.ca/light-pollution/maps/breast-cancer-mortality

**Comment by srkobakian** [comment]
Oh I like the magnifying glass approach! A great way to try to solve a small area problem.

The animation over time is great!

I took them out of the original grid, I think they should be included if we are mentioning them.
Working on reading the references @dicook included now, but happy to do this after.

**Comment by jesse-jesse** [comment]
Do you think they should be in the larger grid or we should have them as a separate figure that shows interaction?

**Comment by srkobakian** [comment]
added them as a new figure, down the bottom, in the animating chunk

**Comment by dicook** [comment]
Interactivity and animation look great!

**Comment by dicook** [comment]
I am a bit concerned that the interactivity section is talking only about publicly available atlases. The figure has examples from atlases that are available. This is ok, but it belongs with the section at the top. And those atlases may need to be included in the list of publicly available atlases described in Section 3.1.

And this section needs to summarise research on interactive and animation tools with maps. And describe how these might be incorporated in new designs. It doesn't need to be very long, but that is the purpose of this section. 

Now, because there are now quite a lot of examples on interactivity, and the section is getting long, it might be a good idea to shift all of this down into section 6, and have two parts to section 6, (1) describing what's done in public atlases, (2) summarise literature on the area and suggest what might be done.

**Comment by srkobakian** [comment]
That sounds great, I will look at that now
